### DISCUSSION TITLE
Welcome to Spine-Toolbox Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [Spine-project/Spine-Toolbox#9] [Discussion]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by olejandro** [comment]
I am Olex. I don't use Spine-Toolbox yet, but I hope to integrate it into my workflow. Great to have a place like this to engage with the community!
